WordPress/Docs/WP/GlobalVariablesOverrideStandard.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Global Variables Override"
+    >
+    <standard>
+    <![CDATA[
+    WordPress global variables should not be overridden.
+
+    Two WordPress global variables, `$content_width` and `$wp_cockneyreplace`, are 
+    specifically intended to be overridden by themes/plugins and are exempt from this rule.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Different variable name.">
+        <![CDATA[
+<em>$my_query</em> = new WP_Query( $args );
+
+
+<em>$GLOBALS['my_data']</em> = "some data";
+        ]]>
+        </code>
+        <code title="Invalid: Global variable overridden.">
```

Comment:
```suggestion
        <code title="Invalid: WordPress global variable overridden.">
```

I think it is important to mention that this is about WordPress globals and not any globals.